lib/public/AppFramework/Utility/ITimeFactory.php

```diff
@@ -58,4 +58,12 @@ public function getDateTime(string $time = 'now', \DateTimeZone $timezone = null
 	 * @since 26.0.0
 	 */
 	public function withTimeZone(\DateTimeZone $timezone): static;
+
+	/**
+	 * @param string|null $timezone
+	 * @return \DateTimeZone
+	 * @throws \Exception
```

Comment:
```suggestion
	 * @throws \DateInvalidTimeZoneException
```
Unless it’s expected to be able to throw other stuff later?

Reply:
But new Timezone throws a simple \Exception?

Reply:
Ouuuh you’re right, I got tricked by documentation, this changed in PHP 8.3, so we have to keep general `\Exception` until 8.2 is EOL ^^.

Reply:
> until 8.2 is EOL

*me cries in 8.0*